### PR TITLE
Adding plumbum recipe

### PR DESCRIPTION
**Pull request by henryiii**
This should add version 1.6.2 of the Plumbum library.

### CODE REVIEW

File: recipes/plumbum/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
```
Comment (synapticarbors) [inline recipes/plumbum/meta.yaml]: Please delete this comment

File: recipes/plumbum/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    remote command execution (over SSH), local and remote file-system paths, easy
+    working-directory and environment manipulation, a programmatic Command-Line
+    Interface (CLI) application toolkit, and a color library.
+  doc_url: https://plumbum.readthedocs.io/
+  dev_url: https://github.com/tomerfiliba/plumbum
+
+extra:
+  recipe-maintainers:
+    - henryiii
+    - tomerfiliba
```
Comment (synapticarbors) [inline recipes/plumbum/meta.yaml]: @tomerfiliba -- Are you ok being added as a co-maintainer on this conda-forge feedstock?

File: recipes/plumbum/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  imports:
+    - plumbum
+    - plumbum.cli
+    - plumbum.colorlib
+    - plumbum.commands
+    - plumbum.fs
+    - plumbum.machines
+    - plumbum.path
+
+  requires:
```
Comment (synapticarbors) [inline recipes/plumbum/meta.yaml]: Unless you are running tests, the simple import checks shouldn't require either of these modules. `paramiko` will be pulled in by the `run` requirement.

Comment (henryiii) [inline recipes/plumbum/meta.yaml]: Okay, thanks, I took them out.

File: recipes/plumbum/meta.yaml (hunk 2 above)
Comment (henryiii) [inline recipes/plumbum/meta.yaml]: Removed @tomerfiliba for now, can add back later

**Pull request merged**